src/lib/getLanguageName.js

```diff
@@ -1,4 +1,5 @@
 const languageNameMap = {
+  de: 'Deutsch',
```

Comment:
Can we move this to the end of the array? The language order is intentional and displayed in the UI.

Reply:
Sure, I thought the order was alphabetical, a closer look would have prevented that. Will change it soon.